docs/resources/rule_set_response_header.md

```diff
@@ -3,19 +3,19 @@ layout: "wallarm"
 page_title: "Wallarm: wallarm_rule_set_response_header"
 subcategory: "Rule"
 description: |-
-  Provides the "Specify a response header" rule resource.
+  Provides the "Change server response headers" rule resource.
 ---
 
 # wallarm_rule_set_response_header
 
 !> The resource requires additional permissions. Ask the support team to obtain them.
```

Comment:
what are these permissions?

Reply:
This note seems to be deprecated too. All roles can manage set_response_header rules.